### DISCUSSION TITLE
Question: how to change character between username & hostname?

### DISCUSSION THREAD

**Discussion by mengzyou** [starship/starship#1244] [General]
Hi, the character between username & hostname is always 'on', like below:  

`mengz on linux-dev in ~`, how can I customize it?  

and also for 'in' before directory?

**Comment by matchai** [comment]
Module prefixes can't currently be changed for all modules. We are in the process of working on a feature to make it so any part of the prompt can be changed. Please follow #624 for updates.

**Reply by mengzyou** [reply]
Okay, thanks for your help!
